src/3box.js

```diff
@@ -131,6 +118,15 @@ class Box extends BoxApi {
     return box
   }
 
+  /**
+   * Returns and 3ID Connect Provider to manage keys, authentication and account links. Becomes default in future.
+   *
+   * @return    {3IDProvider}        Promise that resolves to a 3ID Connect Provider
+   */
+  static get3idConnectProvider () {
+    return threeIdConnect.get3idProvider()
```

Comment:
Should this throw a descriptive error if we're not in a browser?

Reply:
yeah ill add that quickly before merging